src/services/notificationService.ts

```diff
@@ -93,48 +98,78 @@ export default class NotificationService {
             );
           }
         );
-      });
-    });
+      }
+    );
 
   public readonly createOrUpdateInstallation = (
     fiscalCode: FiscalCode,
     installationID: InstallationID,
     installation: Installation
-  ): Promise<IResponseErrorInternal | IResponseSuccessJson<SuccessResponse>> =>
-    withCatchAsInternalError(() => {
-      const notificationHubService = azure.createNotificationHubService(
-        this.hubName,
-        this.endpointOrConnectionString
+  ): Promise<
+    IResponseErrorInternal | IResponseSuccessJson<SuccessResponse>
+  > => {
+    const azureInstallation: IInstallation = {
+      installationId: this.allowMultipleSessions
```

Comment:
isn't this `!this.allowMultipleSessions` ?

Reply:
let's remove `allowMultipleSessions` altogether since we don't need it anymore

Reply:
Fixed in fd811c7. It will be removed in another PR